content/en/docs/components/katib/katib-config.md

```diff
@@ -0,0 +1,211 @@
++++
+title = "Katib Configuration Overview"
+description = "How to make changes in Katib configuration"
+weight = 40
+                    
++++
+
+This page describes information about
+[Katib config](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/katib-config.yaml).
+
+Katib config is the Kubernetes
+[Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
```

Comment:
Thank you for the PR @andreyvelich 

Hi, what do you think of this suggestion? 
- "refactor" to 'This page describes Katib config — the Kubernetes Config Map that contains information about:'

```suggestion
This page describes
[Katib config](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/katib-config.yaml) — 
the Kubernetes
[Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
```

Reply:
Agree, changed.